brackend/db/models.py

```diff
@@ -1,11 +1,21 @@
 """Module that defines/creates/holds ORMs for the database."""
-from brackend.util import BrackendException, DOCKER_POSTGRES_URL
-from sqlalchemy import Boolean, Column, ForeignKey, Integer, String, create_engine
+import enum
+from datetime import datetime
+
+from sqlalchemy import Boolean, Column, DateTime, Enum, ForeignKey, Integer, String, create_engine
 from sqlalchemy.orm import declarative_base, relationship
 
+from brackend.util import DOCKER_POSTGRES_URL, BrackendException
+
 Base = declarative_base()
 
 
+class ProgressEnum(enum.Enum):
```

Comment:
nit: Will Tournaments re-use this and so you're thinking this is just a general `Progress`? Unless we know for sure the sets of statuses used by matches/tournaments/etc will be the same, might want to go with `MatchProgress` here.

nit part 2: we know this is an enum, so you can drop the enum part of the name